### DISCUSSION TITLE
Sioyek

### DISCUSSION THREAD

**Discussion by DerpGusta** [catppuccin/catppuccin#1996] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

There is no specific documentation on colorscheme creation but I'm linking the official settings page and some examples

[Project page](https://sioyek.info/)
[Sioyek Settings](https://sioyek-documentation.readthedocs.io/en/latest/configuration.html)

Examples:
- [Gruvbox theme article](https://mil.ad/blog/2022/gruvbox-for-sioyek.html)
- [Dracula theme for Sioyek](https://github.com/dracula/sioyek/blob/main/dracula.config)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by 21st-centuryman** [comment]
Would love this. Not sure if I have the time to make a port

**Comment by 21st-centuryman** [comment]
I decided to make this into a repo. So far I have only created the mocha variant. If I get the time I will read up on how to actually request that this is in mainline catppuccin

https://github.com/21st-centuryman/Sioyek

EDIT: now I have done them all

**Reply by 21st-centuryman** [reply]
#2166
